### ISSUE TITLE
 [Feature Request] Account switcher

### ISSUE THREAD

**Issue by martienne17** [joelacus/RedditEnhancer#24]
I installed this extension because Reddit Enhancement Suite is no longer being developed or supported. One feature I used to use it for was the account switcher. I was wondering if you would consider adding this feature to your extension.

**Comment by joelacus** [comment]
Hi. I can certainly try to add this feature yes, but I'll need to figure out a secure way of doing it. 
How did it work in RES? Did you have to give it your username and password for each account?

**Comment by martienne17** [comment]
Yes, there was a menu item where you entered the username and password for each. They did have a disclaimer on it that it stored your username and password for the use of this function.

**Comment by joelacus** [comment]
Yes, I'd have to put a disclaimer too. Even though the data saved by the extension is relatively secure, it is not encrypted. However, the data is only saved locally to your machine, but is also synchronised with your Mozilla/Google/Microsoft account if you have browser sync enabled, so no third party should have access. Despite this, I'm not comfortable with storing the username and password in "plaintext", so I'd suggest an option to encrypt the credentials with a pin number or master password to add some level of security.

**Comment by martienne17** [comment]
Understood. I hope you can find the right solution.

**Comment by sneazy-ibo** [comment]
Any updates on this?

**Comment by joelacus** [comment]
Apologies for the delay. I'll work on this next, hopefully I can add it soon.
